encryption/aes_ecb_test.go

```diff
@@ -0,0 +1,59 @@
+package encryption
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAesECB(t *testing.T) {
+	t.Run("Encode/Decode", func(t *testing.T) {
+		cipher, err := NewAesECB([]byte("1234567890123456"))
```

Comment:
Are we always going be reading ASCII alphanumeric characters?

Reply:
This is just a random key, so it can be anything here.

Reply:
Can we add a test that doesn't have alphanumeric characters then?

Reply:
I replaced it with randomly generated key